src/Entity/Transformer.php

```diff
@@ -119,13 +119,17 @@ public function load(SnapshotInterface $snapshot)
         $page->setSite($snapshot->getSite());
         $page->setEnabled($snapshot->getEnabled());
 
-        $content = $this->fixPageContent($snapshot->getContent());
+        $content = $snapshot->getContent();
 
         $page->setId($content['id']);
         $page->setJavascript($content['javascript']);
         $page->setStylesheet($content['stylesheet']);
         $page->setRawHeaders($content['raw_headers']);
-        $page->setTitle($content['title']);
+
+        if (isset($content['title'])) {
```

Comment:
Yeah i guess it'll check, if it's exist and it's not null

Then it's fine

Reply:
Yes! https://3v4l.org/b8qF0